### PR TITLE
forabank.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/Forabank.ru.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!--vip. mismatch
+acquiring. mismatch
+www.bonus. mismatch
+prm. mismatch
+yar. mismatch
+perm. timed out-->
```
Comment (J0WI) [inline src/chrome/content/rules/Forabank.ru.xml]: WFM:
card.forabank.ru

404 on:
ftp.forabank.ru
old.forabank.ru
vip.forabank.ru

Wrong content:
acquiring.forabank.ru
bonus.forabank.ru

Comment (galeksandrp) [inline src/chrome/content/rules/Forabank.ru.xml]: Added.

Comment (J0WI) [inline src/chrome/content/rules/Forabank.ru.xml]: What about `old`and `ftp`?

**Comment by J0WI** [comment]
`vip.forabank.ru` is now listed twice and `ftp.forabank.ru` is not listed.

**Comment by galeksandrp** [comment]
Fixed.

**Pull request merged**